public/docs/ts/latest/guide/npm-packages.jade

```diff
@@ -150,16 +150,16 @@ a(id="dev-dependencies")
   They do not have to be deployed with the production application although there is rarely harm in doing so.
 
   ***[concurrently](https://www.npmjs.com/package/concurrently)*** - 
-  a utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
+  A utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
```

Comment:
I have been told that if the first letter of the next word sounds like a vowel, it is `an`. But I am not english at the end. So let's roll with what the oxford says.